index.html

```diff
@@ -222,7 +222,7 @@ <h5 class="definition"><code><i>[ le&middot;ga&middot;shEr ]</i>   &nbsp;&nbsp;
                                         </svg>
                                     </div>
                                     <h3 class="feature-title">Develop</h3>
-                                    <p class="text-sm">Innovation, creativity, and collaboration should never be lost in translation, that’s why we created Legesher!
+                                    <p class="text-sm">Innovation and creativity should never be lost in translation, that’s why we created Legesher!
```

Comment:
Hey @BlackSou thank you so much for your feedback.

Unfortunately we are unable to merge these changes as innovation, creativity and collaboration are the three main themes of this project. I appreciate you looking into how you can serve this project! Thank you again!